### DISCUSSION TITLE
Cross platform deep linking

### DISCUSSION THREAD

**Discussion by nickrandolph** [unoplatform/uno.extensions#330] [Ideas]
Common scenario for cross platform apps:
- application shares a link eg https://myfancy.app/deeplink/item?id=1
- user clicks on link
- if native app is installed, the link should open the app and open appropriate page
- if no native app, link should open the wasm app in the browser and open the appropriate page

Recipe needs to show:
- how to define url/scheme support for each native app
- how to share deep link
- how to redirect user to the native app from wasm app (consider what Teams does when you open a link in the browser???)

**Comment by assassin316** [comment]
This is definitely needed to support restful deeplinking as shown above.
